functorch/csrc/BatchRulesReduceOps.cpp

```diff
@@ -285,6 +380,10 @@ TORCH_LIBRARY_IMPL(aten, FT_BATCHED_KEY, m) {
   REDUCTION_BOXED(any.dim);
   REDUCTION_BOXED(argmax);
   REDUCTION_BOXED(argmin);
+  VMAP_SUPPORT("cdist", cdist_batch_rule);
+  VMAP_SUPPORT("_euclidean_dist", euclidean_dist_batch_rule);
```

Comment:
Maybe, there is a similar macro to apply `moveBatchDimToFront` on all input tensors ...